comp_loinc/__main__.py

```diff
@@ -0,0 +1,5 @@
+"""Entry point: Calls CLI"""
```

Comment:
Having this file present allows the CLI to be accessible by calling the package directory: `python -m comp_loinc`

```
python -m comp_loinc
Usage: python -m comp_loinc [OPTIONS] COMMAND [ARGS]...
Try 'python -m comp_loinc --help' for help.

Error: Missing command.
```